### DISCUSSION TITLE
little problem

### DISCUSSION THREAD

**Discussion by abdul-raheem-shahzad** [AammarTufail/machinelearning_ka_chilla#8] [Q&A]
@6:52:26      
Sir aap nae aik baat ki 10 hours videos mai  " same file ko excel mai karna hai (kashti) " es Question  ki samj nai aye sirf question samjha de 
aur agar app ka matlab yai hai ke Excel Viewer Extension  ka through open karna hai tou us ke "open with" mai Excel Viewer Extension ka option nai arah hai
![image](https://user-images.githubusercontent.com/85006885/158508234-5bcfcfa2-bbab-4356-85f0-3ef8d63b480c.png)

**Comment by taqihaider7** [comment]
ye csv file ha tu csv editor he use ho ga

**Reply by AammarTufail** [reply]
install an Extension for csv viewer
